packages/grafana-data/src/transformations/transformDataFrame.ts

```diff
@@ -24,11 +27,20 @@ const getOperator =
     const defaultOptions = info.transformation.defaultOptions ?? {};
     const options = { ...defaultOptions, ...config.options };
 
+    const interpolated = isScenes
+      ? options
+      : deepIterate(options, (v) => {
+          if (typeof v === 'string') {
+            return ctx.interpolate(v);
+          }
+          return v;
+        });
```

Comment:
I'm inclined to back off from the in-scenes implementation and have the intepolation done here. There's no benefit (other than micro optimisation of early return if no variable dependency is in placce) of keeping it in scenes knowing that we need to do the same here for the sake of the old arch.

Reply:
my only argument for also keeping it in scenes is so we have less to refactor when we rip the old arch out.

Reply:
this interpolation could be very costly for large transformations configs (which is not all that uncommon), doing it in scenes at least makes sure we do not interpolate it uneasily 

I think it was a mistake to add it here, if they disable scenes it they loose access to new scenes only features that has been true for other features as well